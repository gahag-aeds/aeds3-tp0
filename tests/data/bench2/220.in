220 110
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 60 86 90 8 3 92 27 58 98 39 56 67 67 18 76 82 14 44 49 82 29 91 71 83 57 73 47 25 2 90 70 63 76 61 71 79 5 50 37 56 89 46 23 8 16 52 42 30 48 91 12 77 34 84 12 91 9 59 68 12 49 39 27 77 52 50 56 57 0 46 13 41 92 89 1 60 41 43 90 41 86 55 18 20 39 30 63 0 42 32 12 43 23 39 21 75 41 29 84 41 27 50 34 71 91 35 31 32 78 22 73 16 77 43 36 68 25 0 68 67 32 81 63 7 72 36 82 66 65 18 7 93 68 42 64 59 29 96 43 8 
Add 101 173
Min 139 211
Add 84 156
Max 44 116
Sub 116 188
Min 130 202
Sub 50 122
Sum 30 102
Sub 22 94
Max 47 119
Add 102 174
Min 28 100
Add 28 100
Sum 54 126
Add 48 120
Max 11 83
Sub 114 186
Max 44 116
Sub 139 211
Max 83 155
Add 108 180
Max 29 101
Add 142 214
Sum 63 135
Add 39 111
Max 74 146
Add 35 107
Sum 81 153
Add 143 215
Sum 36 108
Add 27 99
Min 52 124
Add 130 202
Sum 66 138
Sub 51 123
Sum 99 171
Sub 4 76
Min 100 172
Sub 84 156
Max 128 200
Add 147 219
Max 38 110
Sub 111 183
Sum 27 99
Sub 48 120
Max 11 83
Sub 37 109
Sum 117 189
Sub 10 82
Max 131 203
Sub 8 80
Min 118 190
Sub 13 85
Max 2 74
Sub 136 208
Min 7 79
Add 76 148
Max 23 95
Sub 87 159
Sum 27 99
Sub 14 86
Sum 69 141
Sub 79 151
Min 100 172
Add 135 207
Sum 129 201
Add 51 123
Min 94 166
Sub 82 154
Sum 90 162
Add 75 147
Min 37 109
Add 83 155
Max 38 110
Sub 63 135
Min 130 202
Sub 49 121
Max 56 128
Add 56 128
Min 58 130
Sub 39 111
Sum 63 135
Add 28 100
Sum 51 123
Sub 21 93
Max 119 191
Add 58 130
Sum 96 168
Sub 98 170
Sum 144 216
Sub 122 194
Sum 111 183
Sub 22 94
Sum 9 81
Add 63 135
Sum 9 81
Sub 138 210
Sum 69 141
Add 102 174
Min 28 100
Sub 11 83
Sum 132 204
Sub 64 136
Max 50 122
Add 23 95
Min 82 154
Sub 32 104
Max 71 143
Sub 137 209
Min 43 115
