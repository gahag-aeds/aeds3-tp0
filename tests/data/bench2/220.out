0
99
0
3437
97
0
3703
98
99
99
99
3974
100
3956
3772
5
4189
3746
3
95
100
3612
98
3361
95
3
94
-2
99
3339
3888
0
3450
1
3700
-2
99
1
97
-2
3730
3642
94
3673
3573
3309
3006
3025
3892
-5
3349
97
-1
97
-6
